cmd/luna/app/import.go

```diff
@@ -9,8 +9,31 @@ import (
 	"github.com/quay/claircore/libvuln"
 	"github.com/quay/zlog"
 	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
 )
 
+func NewImportCommand() *cobra.Command {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported function NewImportCommand should have comment or be unexported